logos-codegen/src/graph/regex.rs

```diff
@@ -16,6 +16,7 @@ impl<Leaf: Disambiguate + Debug> Graph<Leaf> {
         then: NodeId,
         miss: Option<NodeId>,
         reserved: Option<ReservedId>,
+        repeated: bool,
```

Comment:
I am testing if we can still take the ASCII fast path with dot-repetition, but need to double check my code